.github/workflows/update-specs-and-client-libraries.yaml

```diff
@@ -0,0 +1,133 @@
+name: update-specs-and-client-libraries
+run-name: Update specs and client libraries
+on:
+  push:
+    branches:
+      - master
+    paths-ignore:
+      - 'generated/artifacts/**'
+  pull_request:
+    branches:
+      - master
+  workflow_dispatch:
+    inputs:
+      type-of-change:
+        description: 'Type of change?'
+        required: true
+        default: Patch
+        type: choice
+        options:
+          - Major
+          - Minor
+          - Patch
+      update-onfido-node:
+        description: 'Update onfido-node?'
+        required: true
+        default: false
+        type: boolean
+jobs:
+  generate_specs_and_libraries:
```

Comment:
Perhaps we should have a different job for each build to have more visibility when something fails.

Not thinking about performance because the generation is rather fast.